### PR TITLE
Release v0.13

### PR DESCRIPTION
**Pull request by dvdplm**
Based on #173 – only https://github.com/paritytech/substrate-subxt/pull/175/commits/1c3cf61fc26a0a5d5774acbb126fc0572c95ffc7 is new here.

# Version 0.13.0
* Update to Substrate 2.0.0 [#173](https://github.com/paritytech/substrate-subxt/pull/173)
* Display RawEvent data in hex [#168](https://github.com/paritytech/substrate-subxt/pull/168)
* Add SudoUncheckedWeightCall [#167](https://github.com/paritytech/substrate-subxt/pull/167)
* Add Add SetCodeWithoutChecksCall [#166](https://github.com/paritytech/substrate-subxt/pull/166)
* Improve contracts pallet tests [#163](https://github.com/paritytech/substrate-subxt/pull/163)
* Make Metadata types public [#162](https://github.com/paritytech/substrate-subxt/pull/162)
* Fix option decoding and add basic sanity test [#161](https://github.com/paritytech/substrate-subxt/pull/161)
* Add staking support [#160](https://github.com/paritytech/substrate-subxt/pull/161)
* Decode option event arg [#158](https://github.com/paritytech/substrate-subxt/pull/158)

### CODE REVIEW

**Review by ascjones — APPROVED** [review]
LGTM, just needs master merging.

**Comment by DemiMarie** [comment]
The integration tests fail for me.  I am checking if this is due to an outdated Polkadot.

File: test-node/runtime/src/lib.rs [hunk 1]
```diff
@@ -183,6 +183,8 @@ parameter_types! {
     pub const Version: RuntimeVersion = VERSION;
 }
 
+// const_assert!(AvailableBlockRatio::get().deconstruct() >= AVERAGE_ON_INITIALIZE_WEIGHT.deconstruct());
+
```
Comment (DemiMarie) [inline test-node/runtime/src/lib.rs:187]:
```suggestion
```

Was this intentional?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+* Update to Substrate 2.0.0 [#173](https://github.com/paritytech/substrate-subxt/pull/173)
+* Display RawEvent data in hex [#168](https://github.com/paritytech/substrate-subxt/pull/168)
+* Add SudoUncheckedWeightCall [#167](https://github.com/paritytech/substrate-subxt/pull/167)
+* Add Add SetCodeWithoutChecksCall [#166](https://github.com/paritytech/substrate-subxt/pull/166)
+* Improve contracts pallet tests [#163](https://github.com/paritytech/substrate-subxt/pull/163)
+* Make Metadata types public [#162](https://github.com/paritytech/substrate-subxt/pull/162)
+* Fix option decoding and add basic sanity test [#161](https://github.com/paritytech/substrate-subxt/pull/161)
+* Add staking support [#160](https://github.com/paritytech/substrate-subxt/pull/161)
+* Decode option event arg [#158](https://github.com/paritytech/substrate-subxt/pull/158)
+
```
Comment (DemiMarie) [inline CHANGELOG.md:12]:
```suggestion
* Remove unnecessary Sync bound [#172](https://github.com/paritytech/substrate-subxt/pull/172)

```

**Review by DemiMarie — COMMENTED** [review]
#172 needs to be merged first.

**Comment by dvdplm** [comment]
> The integration tests fail for me. I am checking if this is due to an outdated Polkadot.

Can you be more specific? All of them? How did you run them? One by one? All at once? Which node type did you try and how did you start it, full command?

**Comment by DemiMarie** [comment]
It was due to an old version of polkadot.  The tests pass with git master.

**Comment by DemiMarie** [comment]
@ascjones @dvdplm this cannot be merged until the blockchain has updated to the latest metadata.

**Comment by dvc94ch** [comment]
Don't follow. Which chain do you mean? Polkadot, kusama, edgeware, sunshine or some other chain?

**Comment by DemiMarie** [comment]
@dvc94ch Polkadot and Kusama

**Comment by dvc94ch** [comment]
So are they guaranteed to be updated at the same time? I don't see what the deployment has to do with merging this pr. If you want to treat those chains specially, you can add branches for them, but really it's the responsibility of people building with subxt for a specific chain to make sure they have a combination that works.

**Comment by dvc94ch** [comment]
So are they guaranteed to be updated at the same time? I don't see what the deployment has to do with merging this pr. If you want to treat those chains specially, you can add branches for them, but really it's the responsibility of people building with subxt for a specific chain to make sure they have a combination that works.

**Comment by ascjones** [comment]
> @ascjones @dvdplm this cannot be merged until the blockchain has updated to the latest metadata.

I don't think it's blocker, we don't need to our releases to be in sync with what is deployed on any particular chain. We will make releases which are compatible with new versions of substrate as they come along, and people will choose whichever release matches their target chain.

This will be better now substrate has proper versioning (no longer pre-release), so we can track them.

**Comment by dvdplm** [comment]
> I don't think it's blocker,

Yeah, my bad. I had `ledgeracio` in mind here, not subxt.

**Comment by ascjones** [comment]
I don't know when the upgrade for substrate 2.0 is planned for kusama/polkadot, but it surely is soon. 

In the meantime you could always craft an intermediate release with `*-rc6 + Staking` which would allow you to release `ledgeracio` if that is urgent.

**Pull request merged**